rerun_py/rerun_sdk/rerun/time.py

```diff
@@ -108,35 +108,44 @@ def set_time(
         )
 
 
-def to_nanos(duration: int | float | timedelta | np.timedelta64) -> int:
-    if isinstance(duration, (int, np.integer)):
-        return 1_000_000_000 * int(duration)  # Interpret as seconds and convert to nanos
-    elif isinstance(duration, float):
-        return round(1e9 * duration)  # Interpret as seconds and convert to nanos
+def to_nanos(duration: int | np.integer | float | np.float64 | timedelta | np.timedelta64) -> int:
```

Comment:
this would really benefit from being renamed `second_to_nanos` (assuming it's not published in the `rerun` namespace)

Reply:
it's not only from seconds though, it is also from `timedelta`